packages/client/wallets/aa/src/types/Config.ts

```diff
@@ -33,8 +28,9 @@ export type PasskeySigner = {
 
     /**
      * Displayed to the user during passkey registration or signing prompts.
+     * If not provided, a default name will be used.
      */
-    passkeyName: string;
+    passkeyName?: string;
```

Comment:
It would be good to add a default `walletConfig` to `getOrCreateWallet`, with passkeys set to the default so clients can make calls that look like

```TS
sdk.getOrCreateWallet({ jwt }, "base")
```